skrub/_dataframe/_common.py

```diff
@@ -825,6 +826,26 @@ def _to_categorical_polars(col):
     return _cast_polars(col, pl.Categorical())
 
 
+@dispatch
+def is_all_null(col):
+    raise NotImplementedError()
+
+
+@is_all_null.specialize("pandas", argument_type="Column")
+def _is_all_null_pandas(col):
+    return all(is_null(col))
+
+
+@is_all_null.specialize("polars", argument_type="Column")
+def _is_all_null_polars(col):
+    if col.dtype == pl.Null:
+        return True
+    # col is non numeric
```

Comment:
not sure I understand the comment

Reply:
it was a leftover from the previous version, I rewrote the comments